code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -2965,26 +2965,29 @@
 	if(..())
 		return 1
 
+	var/mob/living/carbon/human/H
```

Comment:
Right, but can you access the `species` var of a human via a macro if the reference you feed it isn't cast as a human? It does `A.species` not `A:species`, so I figured the type cast mattered.